Sources/_NIOFileSystem/Internal/System Calls/FileDescriptor+Syscalls.swift

```diff
@@ -311,13 +331,15 @@ extension FileDescriptor {
     }
 }
 
-#if canImport(Glibc) || canImport(Musl) || canImport(Bionic)
+#if !os(FreeBSD) && (canImport(Glibc) || canImport(Musl) || canImport(Bionic))
```

Comment:
Under what circumstance would this check have previously compiled? Does FreeBSD report its libc as `Glibc`?

I see from the rest of the code that it does. Is there any way we can change that before formal support? It's going to make life _very_ difficult, as well as being factually untrue.

Reply:
Unfortunately FreeBSD does report its libc as Glibc, I do have a PR open (https://github.com/swiftlang/swift/pull/79261) but we have decided that that's no bandwidth to make the overlay change into 6.2.x
cc @etcwilde